src/Analysis/GWASV2/Shared/constants.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/prefer-default-export */
+export const gwasV2Steps = [
+  {
+    title: 'Select Study Population',
+    secondaryTitle: 'Edit Study Population',
+  },
+  {
+    title: 'Select Outcome Phenotype',
+    secondaryTitle: 'Edit Outcome Phenotype',
+  },
+  {
+    title: 'Select Covariate Phenotype',
+    secondaryTitle: 'Edit Covariate Phenotype',
+  },
+  {
+    title: 'Configure GWAS',
+    secondaryTitle: 'Configure GWAS',
+  },
+];
```

Comment:
regarding my previous comments in the unit tests file...maybe use these variables instead of repeating the titles in the tests?

Reply:
Good idea! I imported the constants file and used those variables.